### PR TITLE
Fixed Typographical Error in README.md

### PR DESCRIPTION
**Pull request by iamrajiv**
Fixed Typographical Error in README.md

### CODE REVIEW

**Comment by iamrajiv** [comment]
@asmeurer @sidhantnagpal can you review this PR

File: README.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ This token is currently given on the @sympy-bot GitHub user. To regenerate it,
 login as @sympy-bot, and go to https://github.com/settings/tokens/. Create a
 new token, checking only the `public_repo` box. Be sure to indicate in the
 description that the token is for Travis testing, and be sure to revoke any
-old tokens. The @sympy-bot user needs push access to this repo for the test to
+old tokens. The @sympy-bot user needs to push access to this repo for the test to
```
Comment (certik) [inline README.md]: The `to` shouldn't be here i think.

Comment (iamrajiv) [reply README.md]: yup got it thanks for the suggestion @certik

File: README.md [hunk 2]
```diff
@@ -31,16 +31,16 @@ If the bot stops working, look at the logs with
 
     heroku logs -a sympy-bot
 
-(you need to install the `heroku` command line tools and have access to the
+(you need to install the `Heroku` command line tools and have access to the
```
Comment (asmeurer) [inline README.md]: `heruku` is in backquotes (code) because it refers to the name of the command line tool, which is itself lowercase.

**Comment by iamrajiv** [comment]
is this PR ready to be merged @asmeurer

**Comment by iamrajiv** [comment]
is this PR ready to be merged @asmeurer

File: README.md [hunk 3]
```diff
@@ -78,11 +78,11 @@ to enable the bot version environment variable (run this if the version is
 ## Testing
 
 There is a testing deploy set up at
-https://dashboard.heroku.com/apps/sympy-bot-testing. Currently it is
+https://dashboard.heroku.com/apps/sympy-bot-testing. Currently, it is
 configured to run on https://github.com/asmeurer/GitHub-Issues-Test.
 
 To test, push to a separate branch (`master` has branch protection) on this
-repo (you can also set up a separate testing deploy for your fork if you
+repo (you can also seta up separate testing deploy for your fork if you
```
Comment (sidhantnagpal) [inline README.md]: This line still needs to be addressed, as suggested previously.

Comment (iamrajiv) [reply README.md]: thanks, @sidhantnagpal for suggestion

Comment (iamrajiv) [reply README.md]: thanks, @sidhantnagpal for the suggestion

**Comment by iamrajiv** [comment]
is this ready to be merged @sidhantnagpal

**Pull request merged**